### PR TITLE
Simpler topology

### PR DESCRIPTION
**Pull request by deepfire**
1. Simplify topology file format: `s/NetworkTopology/NodeSetup/`
2. Eliminate `NetworkTopology` and `TopologyInfo` types.
3. Update scripts.

# TODO

- [ ] Update the Chairman test

### CODE REVIEW

**Comment by Jimbo4350** [comment]
Please don't merge this as I've been working on a branch that touches these files.

File: configuration/simple-topology-0.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{ "nodeId": 0
```
Comment (karknu) [inline configuration/simple-topology-0.json]: Why is the nodeId here?

Comment (deepfire) [reply configuration/simple-topology-0.json]: @karknu, updated the format (`NodeSetup` doesn't have `nodeId` anymore), but forgot to remove the field from the actual file -- fixed, thanks!

**Comment by deepfire** [comment]
Getting this now, from the `scripts/shelley-testnet.sh`:
```
cardano-node: ExceptionInLinkedThread "ThreadId 54" prevPointAndBlockNo: block in future CallStack (from HasCallStack):                                                                 │
  error, called at src/Ouroboros/Consensus/NodeKernel.hs:350:17 in ouroboros-consensus-0.1.0.010PELWW33QmDTLPj1CLHWD:Ouroboros.Consensus.NodeKernel
```

**Comment by deepfire** [comment]
Heavy conflicts with https://github.com/input-output-hk/cardano-node/pull/320 ..

**Comment by deepfire** [comment]
Handing this over to @Jimbo4350, as per agreement.

File: configuration/simple-topology-0.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{ "nodeAddress":
```
Comment (karknu) [inline configuration/simple-topology-0.json]: Why is this needed? I thought that we agreed that local ip address should (optionally) be provided through  cli along with port number?

**Comment by deepfire** [comment]
Dead.

**Comment by Jimbo4350** [comment]
@deepfire This isn't dead its being addressed in https://github.com/input-output-hk/cardano-node/pull/335

**Comment by Jimbo4350** [comment]
@deepfire this isn't dead its being addressed in https://github.com/input-output-hk/cardano-node/pull/335

**Comment by Jimbo4350** [comment]
Closing this PR as this issue is being addressed in #335
